src/routes.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
Seems like this file was copied instead of moved since app/routes.js still exists, maybe delete that.

Reply:
ah, good point. I'll fix that!

Reply:
It's gone!